wondering about `make test_python` and `make_test_cmip6cv`:

the approach i took given the instructions
```bash
# conda env creation + checkout code --> good
git clone --recursive git@github.com:PCMDI/cmor && cd cmor
module load conda
conda create -n cmor_dev -c conda-forge \
  libuuid \
  json-c \
  udunits2 \
  hdf5 \
  libnetcdf \
  openblas \
  netcdf4 \
  numpy \
  openssl \
  python \
  pyfive \
  hdf5plugin \
  $CONDA_COMPILERS
conda activate cmor_dev

# configure --> good
export LDSHARED_FLAGS="-shared -pthread"
export PREFIX=$(python -c "import sys; print(sys.prefix)")

./configure \
  --prefix=$PREFIX \
  --with-python \
  --with-uuid=$PREFIX \
  --with-json-c=$PREFIX \
  --with-udunits2=$PREFIX \
  --with-netcdf=$PREFIX \
  --enable-verbose-test

# good
make install
export HDF5_PLUGIN_PATH="$(python -c 'import hdf5plugin; print(hdf5plugin.PLUGINS_PATH)')"

# largely good but did not work
make test
```

first failure is here:
```bash
make test_python
```

error output seems testing clean-up related. removing the directory and rerunning will not fix the problem, so it's a clash of output between tests under `test_python` is my guess:
```bash
python Test/test_cmor_parent_attrs.py
....EEE
======================================================================
ERROR: test_cmip7_warns_on_parent_attributes_without_parent_experiment (__main__.TestCMIP7WithParentAttributes.test_cmip7_warns_on_parent_attributes_without_parent_experiment)
----------------------------------------------------------------------

OSError: [Errno 39] Directory not empty: './CMIP/PCMDI-test-1-0/piControl/r009i000001dp1f30/tos/tavg-u-hxy-sea/gn'

======================================================================
ERROR: test_cmip7_with_parent_attributes (__main__.TestCMIP7WithParentAttributes.test_cmip7_with_parent_attributes)
----------------------------------------------------------------------

OSError: [Errno 39] Directory not empty: './CMIP/PCMDI-test-1-0/historical/r009i000001dp1f30/tos/tavg-u-hxy-sea/gn'

======================================================================
ERROR: test_cmip7_without_parent_attributes (__main__.TestCMIP7WithParentAttributes.test_cmip7_without_parent_attributes)
----------------------------------------------------------------------

OSError: [Errno 39] Directory not empty: './CMIP/PCMDI-test-1-0/piControl/r009i000001dp1f30/tos/tavg-u-hxy-sea/gn'

----------------------------------------------------------------------
Ran 7 tests in 0.579s

FAILED (errors=3)
make[1]: *** [Makefile:157: test_a_python] Error 1
make[1]: Leaving directory '/home/Ian.Laflotte/Working/cmor'
make: *** [Makefile:236: test_python] Error 2
```

perhaps i shouldn't expect this to pass because it's no longer cmip6, but, second failure is here:
```bash
make test_cmip6_cv
```

error output similarly seems testing cleanup related:
```bash
E
======================================================================
ERROR: testCMIP6 (__main__.TestCase.testCMIP6)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/Ian.Laflotte/Working/cmor/Test/base_CMIP6_CV.py", line 32, in tearDown
    self.remove_file_and_directories(filename)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^
  File "/home/Ian.Laflotte/Working/cmor/Test/base_CMIP6_CV.py", line 14, in remove_file_and_directories
    os.rmdir(filename)
    ~~~~~~~~^^^^^^^^^^
OSError: [Errno 39] Directory not empty: 'CMIP6/CMIP6/DCPP/PCMDI/PCMDI-test-1-0/dcppA-hindcast/s2000-r3i1p1f1/Omon/masso/gn/v20260423'

----------------------------------------------------------------------
```

--- a/QUICKSTART.md
+++ b/QUICKSTART.md
@@ -2,6 +2,12 @@
 
 This document provides a set of quick-start instructions for compiling and installing a local checkout of CMOR (Climate Model Output Rewriter)
 from source. CMOR is typically compiled within a Conda/Mamba environment to easily satisfy its C, Fortran, and Python library dependencies.
+It is based on instructions from [PCMDI/cmor3_documentation](https://cmor.llnl.gov/mydoc_cmor3_github/).
+
+
+### Prerequisites
+- `miniforge` installation, as seen in the [PCMDI/cmor3_documentation](https://cmor.llnl.gov/mydoc_cmor3_github/#installing-miniforge).
+- `mamba` or `conda` executables in your `$PATH`
 
 
 ### 1. Obtain the Source Code and CMIP Tables
@@ -18,25 +24,21 @@ development libraries. Set your compiler variables based on your operating syste
 
 **For Linux:**
 ```bash
-# checked, seems good
 export CONDA_COMPILERS="gcc_linux-64 gfortran_linux-64"
 ```
 
 **For Apple Silicon Mac (M1/M2/M3):**
 ```bash
-# not checked
 export CONDA_COMPILERS="clang_osx-arm64 gfortran_osx-arm64"
 ```
 
 **For Intel Mac:**
 ```bash
-# not checked
 export CONDA_COMPILERS="clang_osx-64 gfortran_osx-64"
 ```
 
 Create and activate the `cmor_dev` environment with all dependencies:
 ```bash
-# conda works too, checked
 mamba create -n cmor_dev -c conda-forge \
   libuuid \
   json-c \
@@ -62,19 +64,16 @@ Set your shared linking flags based on your operating system:
 
 **For Linux:**
 ```bash
-# checked, good
 export LDSHARED_FLAGS="-shared -pthread"
 ```
 
 **For Mac:**
 ```bash
-# not checked
 export LDSHARED_FLAGS="-bundle -undefined dynamic_lookup"
 ```
 
 Run the `configure` script, using Python to dynamically find the path to your active Conda environment:
 ```bash
-# checked, linux
 export PREFIX=$(python -c "import sys; print(sys.prefix)")
 ./configure \
   --prefix=$PREFIX \

